packages/jest-circus/src/utils.js

```diff
@@ -242,28 +255,43 @@ export const getTestID = (test: TestEntry) => {
   return titles.join(' ');
 };
 
-const _formatError = (error: ?Exception): string => {
-  if (!error) {
-    return 'NO ERROR MESSAGE OR STACK TRACE SPECIFIED';
-  } else if (error.stack) {
-    return error.stack;
-  } else if (error.message) {
-    return error.message;
+const _formatError = (errors: ?Exception | [?Exception, Exception]): string => {
```

Comment:
```js
?Exception | {error: Exception, asyncError?: Exception}
```
?

Reply:
I tried this, but I failed horribly... Feel free to fix it if you want, my in progress stuff is here: 8a8d362dffb0dab5d6527598c2471bb6edaf0f71